whitelist.json

```diff
@@ -1,1014 +1 @@
-            "gauge.amq.topic.InFlightCount",
-            "gauge.amq.topic.ProducerCount",
-            "gauge.amq.topic.QueueSize",
-            "message.age.average",
-            "message.age.maximum"
-        ]
-    },
-    {
-        "monitorType": "collectd/openstack",
-        "negated": true,
-        "metricNames": [
-            "counter.openstack.nova.server.cpu*",
-            "counter.openstack.nova.server.rx",
-            "counter.openstack.nova.server.rx_packets",
-            "counter.openstack.nova.server.tx",
-            "counter.openstack.nova.server.tx_packets",
-            "gauge.openstack.cinder.limit.maxTotalVolumeGigabytes",
-            "gauge.openstack.cinder.limit.maxTotalVolumes",
-            "gauge.openstack.cinder.limit.totalGigabytesUsed",
-            "gauge.openstack.cinder.limit.totalVolumesUsed",
-            "gauge.openstack.neutron.floatingip.count",
-            "gauge.openstack.neutron.network.count",
-            "gauge.openstack.neutron.router.count",
-            "gauge.openstack.neutron.securitygroup.count",
-            "gauge.openstack.neutron.subnet.count",
-            "gauge.openstack.nova.hypervisor.disk_available_least",
-            "gauge.openstack.nova.hypervisor.free_disk_gb",
-            "gauge.openstack.nova.hypervisor.free_ram_mb",
-            "gauge.openstack.nova.hypervisor.load_average_15m",
-            "gauge.openstack.nova.hypervisor.load_average_1m",
-            "gauge.openstack.nova.hypervisor.load_average_5m",
-            "gauge.openstack.nova.hypervisor.local_gb",
-            "gauge.openstack.nova.hypervisor.local_gb_used",
-            "gauge.openstack.nova.hypervisor.memory_mb",
-            "gauge.openstack.nova.hypervisor.memory_mb_used",
-            "gauge.openstack.nova.hypervisor.running_vms",
-            "gauge.openstack.nova.hypervisor.vcpus",
-            "gauge.openstack.nova.hypervisor.vcpus_used",
-            "gauge.openstack.nova.limit.maxSecurityGroups",
-            "gauge.openstack.nova.limit.maxTotalCores",
-            "gauge.openstack.nova.limit.maxTotalFloatingIps",
-            "gauge.openstack.nova.limit.maxTotalInstances",
-            "gauge.openstack.nova.limit.maxTotalRAMSize",
-            "gauge.openstack.nova.limit.totalCoresUsed",
-            "gauge.openstack.nova.limit.totalInstancesUsed",
-            "gauge.openstack.nova.limit.totalRAMUsed",
-            "gauge.openstack.nova.server.memory",
-            "gauge.openstack.nova.server.memory-actual",
-            "gauge.openstack.nova.server.memory-rss",
-            "gauge.openstack.nova.server.rx_drop",
-            "gauge.openstack.nova.server.tx_drop",
-            "gauge.openstack.nova.server.vda_errors",
-            "gauge.openstack.nova.server.vda_read",
-            "gauge.openstack.nova.server.vda_read_req",
-            "gauge.openstack.nova.server.vda_write",
-            "gauge.openstack.nova.server.vda_write_req"
-        ]
-    }
-]
+[]
```

Comment:
What happens to people that upgrade the agent but still reference this file using their existing configs? Won't their filtering break?

Reply:
Yeah that's true, I guess we will need to leave it.

Reply:
Sorry, I know how badly you wanted to burn it.